packages/asc-ui/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -34,6 +36,9 @@ storiesOf('Atoms/Checkbox', module)
       <Label htmlFor="error" label="Error">
         <Checkbox id="error" error />
       </Label>
+      <Label htmlFor="ref" label="Ref">
```

Comment:
Would it not be more appropriate to add a specific entry to the story? Perhaps more generic documentation can be placed in one of the markdown files.

Reply:
yes I agree

Reply:
which markdown file can I use then?

Reply:
Let's leave the documentation for now, I think we can do this in a separate PR.